src/features/main_menu/_components/login/SignIn.vue

```diff
@@ -91,6 +91,12 @@ export default Vue.extend({
       // eslint-disable-next-line @typescript-eslint/no-this-alias
       const self = this
       Auth.signIn(this.email, this.password)
+        .catch(error => {
+          if (error.name === "UserNotFoundException") {
+            return Auth.signIn(this.email.toLowerCase(), this.password)
+          }
+          throw error
+        })
```

Comment:
Preserve legacy case-sensitive logins. If a case-sensitive match cannot be found, follow up with a case-insensitive login.